distribution/changelog.txt

```diff
@@ -4,6 +4,7 @@
 - Change: [#21529] Classify “Southern Sands”, “Tiny Towers”, “Nevermore Park”, “Pacifica” as expert scenarios.
 - Fix: [#910] Extra viewport does not preserve the location when rotating.
 - Fix: [#21543] Crash with creating a TrackIterator with invalid arguments.
+- Fix: [#21434] Number of guests overflows in objective text.
```

Comment:
This should be one line up (the changelog is ordered by category and then by ID).

Reply:
Fixed.